### PR TITLE
Update help.html

### LINKED ISSUE #12260: Duplicate "there are no beta features" text on help page
Screenshot from 2022-09-24 10-00-37

"Currently, no features are in beta." should be removed from under "How can I get a list of PyPI's IP addresses?" heading.

### PR DESCRIPTION
**Pull request by shivam-0718**
"Currently, no features are in beta." line removed as per the [issue#12260](https://github.com/pypi/warehouse/issues/12260),  issued by @nlhkabu  

Fixes #12260.

### CODE REVIEW

File: warehouse/templates/pages/help.html [hunk 1]
```diff
@@ -874,7 +874,7 @@ <h3 id="ips">{{ ips() }}</h3>
 
         <h3 id="beta-badge">{{ beta_badge() }}</h3>
         <p>{% trans %}When Warehouse's maintainers are deploying new features, at first we mark them with a small "beta feature" symbol to tell you: this should probably work fine, but it's new and less tested than other site functionality.{% endtrans %}</p>
-        <p>{% trans %}Currently, no features are in beta.{% endtrans %}</p>
+
```
Comment (di) [inline warehouse/templates/pages/help.html:877]: This is removing the actual line from the 'beta badge' section, but the issue is about removing the duplicate line from the 'how to get up addresses' section.

**Review by di — CHANGES_REQUESTED** [review]

File: warehouse/templates/pages/help.html (hunk 1 above)
Comment (shivam-0718) [reply warehouse/templates/pages/help.html]: Corrected the mistake 👍

**Review by di — APPROVED** [review]

**Review by miketheman — APPROVED** [review]
Looks good! Need an associated `make translations` to update and remove references to that translated line.

**Comment by shivam-0718** [comment]
@di On exactly looking at the translations documentation in the link https://warehouse.pypa.io/translations.html, what is the error in it, and what do I have to resolve it? 

> @shivam-0718 Please see https://warehouse.pypa.io/development/submitting-patches.html#translations for help on resolving the translations issue here.

**Comment by di** [comment]
I reopened this because I was going to push the translations updates for you, but I don't think I can because you deleted your fork. If you'd like to recreate the fork and make a new PR, I can do this for you to get this merged, but otherwise I'll close this again now.

**Pull request closed without merging**